apps/docusaurus/docs/reference/indexer.md

```diff
@@ -0,0 +1,389 @@
+2. Review the descriptions for the tables for accuracy.
+3. Review the types and descriptions for fields for accuracy.
+4. (Optional) Provide an example use case for the table to the description.
+5. (Optional) Change the date on the deprecation notice if we want to. (I guessed for it, assumed all would have the same date, and applied the notice both to the "deprecation" and the "private" fields - so please correct me where I'm wrong!)
+
+Don't worry about the formatting on whitespace for "Description" and such - I'll run a script afterwards before we merge to fix the whitespacing - Jackson
+
+# Indexer API Reference
+
+The Indexer API allows you to access rich data about tokens, accounts, transactions, and events on-chain using GraphQL queries.
+
+You can interactively explore the data discussed here using Hasura explorers here:
+
+- Mainnet: https://cloud.hasura.io/public/graphiql?endpoint=https://api.mainnet.aptoslabs.com/v1/graphql
+- Testnet: https://cloud.hasura.io/public/graphiql?endpoint=https://api.testnet.aptoslabs.com/v1/graphql
+- Devnet: https://cloud.hasura.io/public/graphiql?endpoint=https://api.devnet.aptoslabs.com/v1/graphql
+
+Or via code at the following endpoints:
+
+- Mainnet: https://api.mainnet.aptoslabs.com/v1/graphql
+- Testnet: https://api.testnet.aptoslabs.com/v1/graphql
+- Devnet: https://api.devnet.aptoslabs.com/v1/graphql
+
+As you explore, you can refer back to these reference docs to understand the tables you are interested in. You should use Ctrl + F to quickly find the right entry for that table.
+
+At the bottom of this page we list the tables which are deprecated, along with any alternatives which may replace them. Make sure to check before building any production infrastructure that the table you are using is not deprecated as there was recently a v2 which dramatically simplified the list of tables.
+
+:::tip
+If you are looking up a table with the `_by_pk` suffix, search for the table name without that suffix. `_by_pk` tables are automatically generated for convenience to allow querying by primary key.
+:::
+
+# Indexer Table Reference
+
+:::tip
+Remember to use Ctrl + F to find the table you are interested in! When in doubt, you may also want to query the Hasura tables above to see examples of the data inside.
+:::
+
+## General
+
+### `account_transactions`
+
+_Has an aggregate view for summary data called `account_transactions_aggregate`_
+
+This table maps accounts and transactions that interact with that account.
+
+| Field                         | Type    | Description                                                                                                |
+| ----------------------------- | ------- | ---------------------------------------------------------------------------------------------------------- |
+| account_address               | String! | This is an Aptos account address. Ex. "0x50bc83f01d48ab3b9c00048542332201ab9cbbea61bda5f48bf81dc506caa78a" |
```

Comment:
Can we note somewhere that this has to be 66 characters. If it's less than 66 characters due to 0s being pruned, we have to prefix with 0, e.g. 
0x1 -> 0x0000...001

Reply:
This applies to any address and "IDs" in this table, such as token_data_id.